src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -18,11 +18,44 @@ under the Apache 2.0 license is conditioned upon your compliance with
 such restriction.
 */
 import { capitalize, chain } from 'lodash'
+import {
+  CUSTOM_RANGE_DATE_OPTION,
+  PAST_24_HOUR_DATE_OPTION,
+  PAST_HOUR_DATE_OPTION,
+  PAST_MONTH_DATE_OPTION,
+  PAST_WEEK_DATE_OPTION
+} from '../../utils/datePicker.util'
+import { AVG, AVG_RAW, RAW_DATA_TOTAL, TOTAL } from '../../constants'
```

Comment:
make constants more specific and not general as `AVG` and `TOTAL`, etc.

Reply:
done.